FWCore/PythonUtilities/python/LumiList.py

```diff
@@ -73,12 +73,12 @@ def __init__(self, filename = None, lumis = None, runsAndLumis = None, runs = No
         if isinstance(runsAndLumis, list):
             queued = {}
             for runLumiList in runsAndLumis:
-                for run, lumis in runLumiList.items():
+                for run, lumis in list(runLumiList.items()):
```

Comment:
Here the earlier use of the `items()` generator would work, and would be more efficient
```suggestion
                for run, lumis in runLumiList.items():
```